lib/evss/aws/configuration.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module EVSS
+  module AWS
```

Comment:
Do we need AWS in the namespace? IMO it's a detail of the connection rather than a defining characteristic of the service, maybe `AWSConfiguration`?

Reply:
Agreed, despite how AWS-centric the project naming is, I don't think the deployment environment should be referenced in our code at all. What if they switch gears and decide to run this in MS Azure instead?

Reply:
I went ahead with `EVSS::AWSConfiguration`.  I agree that having a dedicated namespace is overkill and a poor abstraction.